contrib/platform/src/com/sun/jna/platform/win32/Crypt32.java

```diff
@@ -366,7 +367,7 @@ boolean CertGetCertificateChain(HCERTCHAINENGINE hChainEngine, CERT_CONTEXT pCer
      * terminating null character. If psz is NULL or csz is zero, returns the
      * required size of the destination string.
      */
-    int CertNameToStr(int dwCertEncodingType, DATA_BLOB pName, int dwStrType, Pointer psz, int csz);
+    int CertNameToStr(int dwCertEncodingType, DATA_BLOB pName, int dwStrType, NativeWin32String psz, int csz);
```

Comment:
Not needed, `Pointer` is handled properly.  If you need to handle Unicode, do it in a util... which has been done.